security/kubernetes-default-deny.md

```diff
@@ -92,6 +93,140 @@ spec:
   - Egress
 ```
 
+### {{site.prodname}} best practice policy template
```

Comment:
I think we need to add something to the earlier "Enable default deny Calico global network policy, non-namespaced" section. Basically flag there that default deny using that policy can stop k8s or calico control plane working, and applies to all host endpoints too.  Then explain it is usually better to make the policy more targeted (so it doesn't apply to control plane or host endpoints) using a second policy example that is first "deny-all" policy (with tabs). I would rename it that policy from "deny-all" to "default-app-policy" to better reflect what it is.

I don't think we need to define the policies that would secure Calico or k8s control plane components in this doc, but we should mention that best practice you would define these as part of cluster setup before starting any app workloads.

Reply:
I feel like my last commit should cover all these points. #286e2f7
Please let me know if I'm missing anything.